.github/workflows/ci.yml

```diff
@@ -469,6 +469,35 @@ jobs:
       - name: "Validate global Python install"
         run: python3 scripts/check_system_python.py --uv ./uv --externally-managed
 
+  system-test-macos-aarch64-homebrew:
+    needs: build-binary-macos-aarch64
+    name: "check system | homebrew python on macos aarch64"
+    runs-on: macos-14
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Install Python"
+        run: |
+          brew install python@3.8
+
+          # The system provides `python3` and brew only adds `python3.8` to the path so
+          # we need to link it to take precedence over the the default system python
+          ln -s "/opt/homebrew/bin/python3.8" "/opt/homebrew/bin/python3.3"
```

Comment:
Alternatively, we could just `brew install python3` and that will be on the path. Probably cleaner unless we actually care about Python 3.8 coverage? @charliermarsh

Reply:
Would `brew link --force python@3.8` help instead of a manual symlink?